packages/git-butler-deb/git-butler-deb.pacscript

```diff
@@ -0,0 +1,19 @@
+name="git-butler-deb"
+name="git-butler-deb"
+# there is no repology property, since the package is not even in repology (with the exception of an AUR package, with a name that doesnt match the official deb file
+arch=('amd64')
+relpkgver="0.10.6"
+secpkgver="619"
+pkgver="${relpkgver}-${secpkgver}"
```

Comment:
Merge these.

Reply:
Done. I also, in the url property, will use an expression to get the 1st part of pkgver instead of using relpkgver to get it.